ddtrace/settings/config.py

```diff
@@ -17,6 +19,49 @@
 log = get_logger(__name__)
 
 
+def _parse_propagation_styles(name, default=PROPAGATION_STYLE_DATADOG):
```

Comment:
```suggestion
def _parse_propagation_styles(name, default):
```

Don't see a reason why we should provide a default here.